frontend/src/pages/member/LoginPage.vue

```diff
@@ -73,12 +73,31 @@ export default {
         return;
       }
       try {
-        // TODO login API를 작성해주세요.
-        // const { email, password } = this.member;
-        // const data = await fetch("/login")
-        // TODO member 데이터를 불러와 주세요.
-        // const member = wait fetch("/members/me")
-        // this.setMember(member);
+        // TODO login API를 작성해주세요. (v)
+        const { email, password } = this.member;
+        const data = await fetch("http://localhost:8080/login/token", {
```

Comment:
`http://localhost:8080` 이 정보는 환경에 따라 변경가능성이 있어서 보통 Path만 사용해요 :) 
```suggestion
        const data = await fetch("/login/token", {
```

Reply:
상대 경로로 변경해주었습니당당 😁